src/main/java/org/smartregister/pathevaluator/dao/TaskDao.java

```diff
@@ -47,4 +47,6 @@
 	org.smartregister.domain.Task updateTask(org.smartregister.domain.Task task);
 
 	List<Task> findTasksByJurisdiction(String jurisdiction, String planIdentifier);
+
+	List<Task> findTasksByJurisdiction(String jurisdiction);
```

Comment:
Can we use the existing method and pass planIdentifier as `null or empty `and modify its implementation on the server-core to ensure backward compatibility. We also need to check and go with this approach only if it does not impact client-core?

Reply:
I think we can do that

Reply:
Great @ekigamba . Please go ahead with this approach.